forms-flow-web/src/components/ServiceFlow/details/TaskHeaderListView.js

```diff
@@ -0,0 +1,362 @@
+import React, { useEffect, useState } from "react";
+import { Row, Col } from "react-bootstrap";
+import {
+  getISODateTime,
+  getFormattedDateAndTime,
+  getProcessDataObjectFromList,
+} from "../../../apiManager/services/formatterService";
+import { useDispatch, useSelector } from "react-redux";
+import DatePicker from "react-datepicker";
+import moment from "moment";
+import "react-datepicker/dist/react-datepicker.css";
+import "./../ServiceFlow.scss";
+import AddGroupModal from "./AddGroupModal";
+import {
+  claimBPMTask,
+  // fetchFilterList,
+  fetchServiceTaskList,
+  getBPMTaskDetail,
+  unClaimBPMTask,
+  updateAssigneeBPMTask,
+  updateBPMTask,
+} from "../../../apiManager/services/bpmTaskServices";
+import { setBPMTaskDetailUpdating } from "../../../actions/bpmTaskActions";
+//import UserSelection from "./UserSelection";
+import UserSelectionDebounce from "./UserSelectionDebounce";
+import SocketIOService from "../../../services/SocketIOService";
+import { useTranslation } from "react-i18next";
+
+const TaskHeaderListView = React.memo(({task,taskId,groupView = true}) => {
+  // const task = useSelector((state) => state.bpmTasks.taskDetail);
+  // const taskId = useSelector((state) => state.bpmTasks.taskId);
+  const processList = useSelector((state) => state.bpmTasks.processList);
+  const username = useSelector(
+    (state) => state.user?.userDetail?.preferred_username || ""
+  );
+  const taskGroups = useSelector((state) => state.bpmTasks.taskGroups);
+  const selectedFilter = useSelector((state) => state.bpmTasks.selectedFilter);
+  const reqData = useSelector((state) => state.bpmTasks.listReqParams);
+  const firstResult = useSelector((state) => state.bpmTasks.firstResult);
+  const [followUpDate, setFollowUpDate] = useState(null);
+  const [dueDate, setDueDate] = useState(null);
+  const [showModal, setModal] = useState(false);
+  const [isEditAssignee, setIsEditAssignee] = useState(false);
+  const dispatch = useDispatch();
+  const { t } = useTranslation();
+  useEffect(() => {
+    console.log(task, taskId);
```

Comment:
remove console

Reply:
@fahad-aot  Please check any comments coming to this PR and resolve

Reply:
okay